Tests/MoyaSugarTests/MoyaSugarProviderTests.swift

```diff
@@ -24,7 +24,7 @@ class MoyaSugarProviderTests: XCTestCase {
         XCTAssertEqual($0.sugar.url, "https://api.github.com/user/devxoul/repos?page=2")
         XCTAssertEqual($0.moya.method, $0.sugar.method)
         XCTAssertEqual(String(describing: $0.moya.parameters), String(describing: $0.sugar.parameters))
-        XCTAssertEqual($0.sugar.parameterEncoding is URLEncoding, true)
+        XCTAssertNil($0.sugar.parameterEncoding)
```

Comment:
Why is this behavior changed? How can we test the parameter encoding?

Reply:
It doesn't have parameterEncoding because there's no `parameters`.

```swift
public enum Task {
    /// A request with no additional data.
    case requestPlain

    /// A requests body set with encoded parameters.
    case requestParameters(parameters: [String: Any], encoding: ParameterEncoding)
    ...
}


```

Reply:
Then we can test the enum case of `task`.